Startup strategy configuration in OpenShift Operator (#175)
mramendi/kie-docs: doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

====
If you use RH-SSO or LDAP authentication, the same user must be configured in your authentication system with the `kie-server,rest-all,admin` roles for {PRODUCT}.
====
. Optional: Select the startup strategy. The `OpenShiftStartupStrategy` setting is enabled by default.
+

> In some authoring environments, you might need to ensure that several users can deploy services on the same {KIE_SERVER} at the same time. By default, after deploying a service onto a {KIE_SERVER} using {CENTRAL}, the user needs to wait for some seconds before more services can be deployed. The `OpenShiftStartupStrategy` setting is enabled by default and causes this limitation. To remove the limitation, you can select the `ControllerBasedStartupStrategy` setting from the *Startup Strategy* list.

Reviewer: ```suggestion
In some authoring environments, you might need to ensure that several users can deploy services on the same {KIE_SERVER} at the same time. By default, after deploying a service onto a {KIE_SERVER} using {CENTRAL}, the user must wait a few seconds before more services can be deployed. The `OpenShiftStartupStrategy` setting is enabled by default and causes this limitation. To remove the limitation, select the `ControllerBasedStartupStrategy` setting from the *Startup Strategy* list.
```

Reviewer: If 'a few' is too specific then I suggest 'the user must wait before...
